scripts/add-at-macro/add_at_macro.clj

```diff
@@ -539,18 +542,44 @@
 
       :else
       (doseq [file filter-valid-files]
-        (read-write-file file {:verbose? print?
+        (read-write-file file {:verbose? verbose?
                                :testing? testing?})))))
 
 (defn -main
   "Call this function with the directory path as an argument to run this script. This function
-  is called after `bb core.clj` with the arguments passed to bb. Also see `run-script` above."
-  [& args]
-  (let [directory (str (ffirst args))]
-    (if (seq directory)
-      (run-script directory)
-      (println "Directory/File not provided")))
+  is called after `bb add_at_macro.clj` with the arguments passed to bb.
+  Extra options that can be passed to the command apart from the directory/file are
+   1. `--verbose` - When this is passed, the changes the script makes are printed to console
+   2. `--testing` - When this is passed, the files the script edits are not saved to disk with the new changes
+      but printed to the console.
+  When `--testing` is passed, `--verbose` is always true.  Also see `run-script` above."
+  [directory {:keys [verbose? testing?]}]
+  (if (seq directory)
+    (run-script directory {:verbose? verbose?
+                           :testing? testing?})
+    (println "Directory/File not provided"))
   (System/exit 0))
 
-(when (= *file* (System/getProperty "babashka.file"))
-  (-main *command-line-args*))
+(def usage "Add at Macro Script
+
```

Comment:
We should probably add a description while we are here... 
```
Add at Macro Script

Recursively traverse all the ClojureScript files in an existing codebase, adding `:src (at)` to every use of a re-com component. Where necessary, it will also modify namespace requires to refer the `re-com.core/at` macro.